app/models/manager_refresh/inventory_collection_default/cloud_manager.rb

```diff
@@ -0,0 +1,226 @@
+class ManagerRefresh::InventoryCollectionDefault::CloudManager < ManagerRefresh::InventoryCollectionDefault
+  class << self
+    def vms(extra_attributes = {})
```

Comment:
attributes reminds me of the model attributes
but this is rather params for creating the `InventoryCollections` right?